### DISCUSSION TITLE
data/models/mpiigaze/resnet_preact/model.pth

### DISCUSSION THREAD

**Discussion by wsl19980102** [hysts/pytorch_mpiigaze#38] [Q&A]
hi hysts：
I want to know how this document can be obtained. Is it convenient for you to talk about it？
data/models/mpiigaze/resnet_preact/model.pth
Thank U

**Comment by hysts** [comment]
Hi, @wsl19980102 

You mean you want to run the demo without training the model? If so, it's easy to use [this repo](https://github.com/hysts/pytorch_mpiigaze_demo).

**Reply by wsl19980102** [reply]
I'm sorry I didn't make it clear. I want to train and demonstrate, but I can't find the file when I demonstrate according to the default parameters. I don't know why.

**Reply by wsl19980102** [reply]
I don't think I'm smart enough. I tried to redo your project, but it didn't succeed. There were a lot of problems that I couldn't understand. If you don't mind, I'd like to see your source file. You can contact me through this email or send me the whole file. 1245586112@qq.com

**Reply by hysts** [reply]
@wsl19980102 

As written in [demo section](https://github.com/hysts/pytorch_mpiigaze#demo) of REAME, you need to specify the model path and the path of the camera calibration results in [the configuration file](https://github.com/hysts/pytorch_mpiigaze/blob/master/configs/demo_mpiigaze_resnet.yaml#L6-L7) before running
```bash
python demo.py --config configs/demo_mpiigaze_resnet.yaml
```

Also, all the code needed to run the demo is included in this repo.

**Comment by Prashant-THRSL** [comment]
Here's the Link:
https://github.com/hysts/pytorch_mpiigaze_demo/releases/tag/v0.1.0

And do rename the model **mpiigaze_resnet_preact.pth** to model.pth

Cheers!
